content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+
+#### Sample Agenda & Prompts - 
+1. Arrange the board:
+     - Label the board to identify a name for the process being explored
+     - From the main storming board, identify a selection of key events that describe an end to end process within the larger system and copy them to another board. 
+       - Preconditions: Identify events which need to have happened before this process can begin and put them in a grayed area on the left with the title Preconditions
+       - Postconditions: Identify at least one suitable event which describes the optimal outcome of the process
+       - Arrange any remaining events in a chronological order from left to right between the preconditions and optimal outcomes. Do not worry about accuracy, as the events are narrated, be prepared to change, add or remove events
+     - Add a grammar guide as shown  ![This is the placeholder for process modelling grammar](/images/practices/event-storming-process-modelling/process-modelling-grammar.png)
+
+1. First Narration: Attempt to narrate a `golden path` through the board, describing the process of events as they happen from left to right. Adjust and add grammar as necessary to reach a consensus of the process.
+    - Roles: 
+       - There are typically 3 roles; one Narrater, one Scribe and the remaining participants are The Audience
+         - A Narrator reads through the process from left to right
+         - The Audience listens to the narrator and assists in discovering the process
+         - A Scribe records the process on the board
+       - Any participant can take on any of these roles and it should be encouraged for roles to rotate
+
+    - Questions: If a question cannot be answered straight away then leave it on the board and return to it later.
+    - Grammar: Add grammar appropriately but do not get bogged down by it
+      1. Events: These represent events that are "Interesting" to the wider system. When understanding what is "interesting", you might refer to the description given by Martin Fowler in his article [What do you mean by “Event-Driven”?](https://martinfowler.com/articles/201701-event-driven.html)
+      1. Policies: This sticky represents a business decision to do something, sometimes this is written as a short sentence or can be labeled and described somewhere else. Any description should include the `event` they are listening to an `actor` if one is required, a `business decision` to be made and an `action` to take
+      1. Actors: This sticky represents a human interaction and should be placed with policies to identify who is responsible of making the decision and performing the task, a policy without an actor is automated
+      1. Action: An action that must be performed due to policy and produces one or more interesting events
+      1. Read Models: Data required to make decisions or perform commands
```

Comment:
Adding examples could help to better comprehend what is meant for each definition.